extension/manifest.json

```diff
@@ -19,7 +19,9 @@
     "default_popup": "popup.html"
   },
   "permissions": [
-    "activeTab"
+    "activeTab",
+    "cookies",
+    "https://www.tv4play.se"
```

Comment:
Om vi hade behövt dessa så skulle vi stoppa dem i `optional_permissions` och sedan be användaren att acceptera dem när de försöker använda tillägget på TV4. Detta är enkelt, exempel:

https://github.com/stefansundin/privatkopiera/blob/7d1c8c1ed57ee6bd19e55d17d9dd19db15dd76cc/extension/js/nrk.js#L65-L67